src/components/StaticContent/HowItWorks.re

```diff
@@ -96,4 +96,4 @@ module HowItWorks = {
         </Rimble.Flex>
     </Rimble.Box>;
   };
-};
+};
```

Comment:
It is a small thing, but I prefer to have trailing new lines on files (makes it quicker to edit them, and looks nicer IMO). It is a simple setting in your editor.

Reply:
Cool, setting changed.